conf/sample.ini

```diff
@@ -551,6 +551,21 @@
 # Syslog tag. By default, the process' argv[0] is used.
 ;tag =
 
+[log.frontend]
+
+# Should Sentry be initialized
+;enabled = false
+
+# Sentry DSN if you wanna send events to Sentry. In this case, set custom_endpoint to empty
+;sentry_dsn =
+
+# Custom endpoint to send Sentry events to. If this is configured, DSN will be ignored and events push to this endpoint. Default endpoint will log frontend errors to stdout.
+;custom_endpoint = /log
+
+# Rate of events to be reported between 0 (none) and 1 (all), float
+;sample_rate = 1.0
+
+
```

Comment:
Nit, but could you remove this extra blank line? It also isn't in conf/defaults.ini.

Reply:
- [ ] remove blank line